index.js

```diff
@@ -46,7 +48,18 @@ async function sendQuadStream ({ defaultMediaType, formats, options, quadStream,
   })
 }
 
-function init ({ factory = rdf, formats = defaultFormats, defaultMediaType } = {}) {
+function init ({ factory = rdf, formats = defaultFormats, defaultMediaType, baseIriFromRequest } = {}) {
+  let getBaseIri
+  if (baseIriFromRequest === true) {
+    getBaseIri = (req) => {
+      if (!req.absoluteUrl) absoluteUrl.attach(req)
```

Comment:
No need for the `if`, `.attach` does that already for you, for the case that you have already a `absolute-url` instance in the routing with custom options.